torch/fx/experimental/proxy_tensor.py

```diff
@@ -163,19 +184,77 @@ def extract_val(val):
     elif isinstance(val, (int, float, bool)):
         return val
 
+class PythonKeyTracer(Tracer):
+    def __init__(self):
+        super().__init__(autowrap_modules=())
+        self.tensor_tracker = WeakTensorKeyDictionary()
+        self.symnode_tracker = _SymNodeDict()  # type: ignore[var-annotated]
+        self.script_object_tracker = WeakIdKeyDictionary(dict=None, ref_type=_WeakHashRef)
+        self.traced_func: List[Tuple[str, str]] = []
```

Comment:
Can you document this field?

Reply:
There is another unfortunate divergence behaviour where make_fx could be using https://github.com/pytorch/pytorch/blob/main/torch/fx/experimental/proxy_tensor.py#L839 followiung tracer. You might need to update this too.